src/System.Runtime/tests/System/DoubleTests.cs

```diff
@@ -470,5 +387,101 @@ public static void Parse_Invalid(string value, NumberStyles style, IFormatProvid
             }
         }
+
+        [Fact]
+        public static void PositiveInfinity()
+        {
+            Assert.Equal(1.0 / 0.0, double.PositiveInfinity);
+            Assert.Equal(0x7FF00000_00000000u, DoubleToUInt64Bits(double.PositiveInfinity));
+        }
+
+        public static IEnumerable<object[]> ToString_TestData()
+        {
+            yield return new object[] { double.MinValue, "G", null, "-1.79769313486232E+308" };
+            yield return new object[] { -4567.0, "G", null, "-4567" };
+            yield return new object[] { -4567.89101, "G", null, "-4567.89101" };
+            yield return new object[] { 0.0, "G", null, "0" };
+            yield return new object[] { 4567.0, "G", null, "4567" };
+            yield return new object[] { 4567.89101, "G", null, "4567.89101" };
+            yield return new object[] { double.MaxValue, "G", null, "1.79769313486232E+308" };
+
+            yield return new object[] { double.Epsilon, "G", null, "double.Epsilon" };
+            yield return new object[] { double.NaN, "G", null, "NaN" };
+
+            yield return new object[] { 2468.0, "N", null, "2,468.00" };
+
+            // Changing the negative pattern doesn't do anything without also passing in a format string
+            var customNegativePattern = new NumberFormatInfo() { NumberNegativePattern = 0 };
+            yield return new object[] { -6310.0, "G", customNegativePattern, "-6310" };
+
+            var customNegativeSignDecimalGroupSeparator = new NumberFormatInfo()
+            {
+                NegativeSign = "#",
+                NumberDecimalSeparator = "~",
+                NumberGroupSeparator = "*"
+            };
+            yield return new object[] { -2468.0, "N", customNegativeSignDecimalGroupSeparator, "#2*468~00" };
+            yield return new object[] { 2468.0, "N", customNegativeSignDecimalGroupSeparator, "2*468~00" };
+
+            var customNegativeSignGroupSeparatorNegativePattern = new NumberFormatInfo()
+            {
+                NegativeSign = "xx", // Set to trash to make sure it doesn't show up
+                NumberGroupSeparator = "*",
+                NumberNegativePattern = 0,
+            };
+            yield return new object[] { -2468.0, "N", customNegativeSignGroupSeparatorNegativePattern, "(2*468.00)" };
+
+            NumberFormatInfo invariantFormat = NumberFormatInfo.InvariantInfo;
+            yield return new object[] { double.Epsilon, "G", invariantFormat, "double.Epsilon" };
+            yield return new object[] { double.NaN, "G", invariantFormat, "NaN" };
+            yield return new object[] { double.PositiveInfinity, "G", invariantFormat, "Infinity" };
+            yield return new object[] { double.NegativeInfinity, "G", invariantFormat, "-Infinity" };
+        }
+
+        [Fact]
+        public static void Test_ToString()
+        {
+            RemoteInvoke(() =>
```

Comment:
Why `RemoteInvoke`?

Reply:
It's the way the test was already setup. Same goes for some of the other "weirdness" that the tests have.

Reply:
Because it is changing CurrentCulture which is not thread specific on UWP (although I thought I saw mention of them fixing that)